types/system-logger/index.d.ts

```diff
@@ -0,0 +1,53 @@
+// Type definitions for system-logger 2.1
+// Project: https://github.com/leocwlam/system-logger
+// Definitions by: Leo Lam <https://github.com/leocwlam>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.1
+
+/// <reference types="node" />
+
+export enum level {
```

Comment:
Note that this will allow variables of type `level` to be indexed with numbers (e.g. `[0]` will return `"error"`), which won't work at runtime.